python/practices/practice-3/src/constants/constants.py

```diff
@@ -13,3 +13,9 @@
 
 # Constant representing the number of top words to display in the results.
 TOP_COUNT = 20
+
+# Constant for condition number
+CONDITION_NUMBER = 36
```

Comment:
I don't understand the meaning of this variable name.
I see you use it to add a number and compare the result with another number. So I suggest changing the name.
Here's my suggestion: `VALUE_DIFFERENCE`, find another one.

Reply:
Done